src/routes/load/container/Load.tsx

```diff
@@ -16,17 +16,31 @@ import { SAFELIST_ADDRESS } from 'src/routes/routes'
 import { buildSafe } from 'src/routes/safe/store/actions/fetchSafe'
 import { history } from 'src/store'
 import { loadFromStorage } from 'src/utils/storage'
+import { Dispatch } from 'redux'
+import { SafeOwner, SafeRecordProps } from '../../safe/store/models/safe'
+import { List } from 'immutable'
 
-export const loadSafe = async (safeName, safeAddress, owners, addSafe) => {
+type RecordPropType = Partial<SafeRecordProps>
+
+type StoredSafesType = {
```

Comment:
type is still incorrect, they get stored as a JSON string and `SafeRecordProps` uses immutable.js data scructures like `List` and `Set`

Reply:
now?

Reply:
Sorry, I didn't really understand how removing types fixes the issue